internal/util/version/version.go

```diff
@@ -55,12 +58,25 @@ func init() {
 		return
 	}
 
+	info.BuildEnvironment = types.MustMakeDocument()
 	for _, s := range buildInfo.Settings {
+		info.BuildEnvironment.Set(s.Key, s.Value)
 		switch s.Key {
 		case "vcs.revision":
 			info.Commit = s.Value
 		case "vcs.modified":
 			info.Dirty, _ = strconv.ParseBool(s.Value)
+		case "-race":
+			if raceEnabled, _ := strconv.ParseBool(s.Value); raceEnabled {
+				info.IsDebugBuild = true
+			}
+		case "-tags":
+			// TODO: replace for slices.Contains()
```

Comment:
We totally could that now :)

Reply:
Yeah, just wasn't sure about depending on `golang.org/x/exp` only to use `slices.Contains()` in this place